physionet-django/events/views.py

```diff
@@ -7,12 +7,17 @@
 from django.contrib.auth.decorators import login_required
 from django.forms import modelformset_factory
 from django.urls import reverse
+from django.db import transaction
+from django.utils import timezone
 
 
 import notification.utility as notification
-from events.forms import AddEventForm, EventApplicationResponseForm
-from events.models import Event, EventApplication, EventParticipant
+from project import utility
```

Comment:
For consistency with the other imports (and for clarity with all of the `utility` modules), I think it would be better to do: `from project.utility get_form_errors`.